actions/install/installer.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# SLSA verifier install script
+
+verifier_release="$1"
+install_dir="$HOME/.slsa/bin/${verifier_release}"
+
+shopt -s expand_aliases
+if [ -z "$NO_COLOR" ]; then
+  alias log_info="echo -e \"\033[1;32mINFO\033[0m:\""
+  alias log_error="echo -e \"\033[1;31mERROR\033[0m:\""
+else
+  alias log_info="echo \"INFO:\""
+  alias log_error="echo \"ERROR:\""
+fi
+set -e
+
+mkdir -p ${install_dir}
+cd ${install_dir}
+
+if [[ "${verifier_release}" == "main" ]]; then
+    log_info "installing verifier via 'go install' from its main version"
+    gobin=$(go env GOPATH)/bin
+    go install github.com/slsa-framework/slsa-verifier/cli/slsa-verifier@main
+    ln -s ${gobin}/slsa-verifier ${install_dir}/slsa-verifier
+    echo "Installed SLSA verifier at ${install_dir}/slsa-verifier"
+    exit 0
+fi
+
+# Download v1.2.0 for bootstrapping
+binary_addr="https://github.com/slsa-framework/slsa-verifier/releases/download/v1.2.0/slsa-verifier-linux-amd64"
+sha256="37db23392c7918bb4e243cdb097ed5f9d14b9b965dc1905b25bc2d1c0c91bf3d slsa-verifier-linux-amd64"
+if [[ $verifier_release == "test-force-checksum-error" ]]; then
+  binary_addr="https://github.com/slsa-framework/slsa-verifier/releases/download/v0.0.1/slsa-verifier-linux-amd64"
+fi
+curl -sL ${binary_addr} -o slsa-verifier-linux-amd64
+echo ${sha256} >> SHA256SUM.md
+
+if ! sha256sum --quiet -c SHA256SUM.md ; then
```

Comment:
it would be nice if you could try to re-use the slsa-github-generator action (https://github.com/slsa-framework/slsa-github-generator/tree/main/.github/actions/compute-sha256) as part of the composite actions, since the sha256sum command is hard to read